src/OpenSSL/__main__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function
+
+import sys
+
+import OpenSSL.SSL
+import cryptography
+
+from . import version
+
+
+if __name__ != "__main__":
+    raise ImportError("This module can't be imported.")
```

Comment:
This code should go in a main function which is invoked in the `__main__` block, that way it can be tested in memory